### ISSUE TITLE
Stubs do not work

### ISSUE THREAD

**Issue by valis** [JetBrains/intellij-arend#64]
It seems that stubs for PSI elements are never created.

**Comment by ice1000** [comment]
Is it created now?

**Comment by valis** [comment]
No, they're not. I think this might be because of serialized core expressions. When they're loaded, it looks at PSI elements which might cause creating the whole element instead of a stub. So, we need to fix loading of serialized data first.

Also, I think they are not created even when there are no serialized data, but I'm not sure. If this is the case, then there are some other issues.

**Comment by ice1000** [comment]
Ok

**Comment by ice1000** [comment]
According to @sxhya, the "import when selecting completion item" feature relies on the stub index, but they do work. Does that mean stub indices are created now?

**Comment by valis** [comment]
Indices do work, but stubs are never created, instead actual PSI elements are always created (not their stubs).
